docs/config.yaml

```diff
@@ -110,6 +108,9 @@ s3:
   # OPTIONAL: object ACL to use when uploading
   acl: private
 
+# OPTIONAL: whether to generate a release index
+release_index: true
```

Comment:
maybe we can add this under a new `misc` section and collect other loose items under that? I had started to do this in https://github.com/coreos/fedora-coreos-pipeline/pull/759 (see [this](https://github.com/coreos/fedora-coreos-pipeline/compare/35cfa7dc3cd8c7052eedf7eb27079b654f60c1bb..cb1cfa71d8cc56f8fba0fe79c87d8a451f0937e6) force push), but we ended up going a different way.

TL;DR - I think things are starting to get a bit disorganized and having a misc "things that don't fit anywhere else" section might be nice.